geometry-in-ruby.gemspec

```diff
@@ -3,12 +3,13 @@ $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
   s.name        = "geometry-in-ruby"
-  s.version     = '0.0.1'
+  s.version     = '0.0.6'
```

Comment:
Included is a commit that updates from 0.0.1 to 0.0.5.  I don't know why our repo doesn't have this code.  I couldn't find it and copied all the files over from the local rubygems location and the commit is the diff.

Reply:
Without this commit with the updated code, dxf fails with missing `options`.  This commit adds in `options`.